Tests/WdUnitTestCase.php

```diff
@@ -14,6 +14,8 @@
 use OxidEsales\Eshop\Core\DatabaseProvider;
 
 use Psr\Log\LogLevel;
+use PHPUnit_Framework_TestResult;
```

Comment:
@ps-cgrach please also remove the **unused imports**

Reply:
👍